code/modules/antagonists/heretic/status_effects/debuffs.dm

```diff
@@ -214,3 +214,90 @@
 /datum/status_effect/heretic_lastresort/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_IGNORESLOWDOWN, TRAIT_STATUS_EFFECT(id))
 	owner.AdjustUnconscious(20 SECONDS, ignore_canstun = TRUE)
+
+
+
+/// Used by moon heretics to make people mad
+/datum/status_effect/moon_converted
+	id = "moon converted"
+	alert_type = /atom/movable/screen/alert/status_effect/moon_converted
+	duration = -1
+	status_type = STATUS_EFFECT_REPLACE
+	///used to track damage
+	var/damage_sustained = 0
+	///overlay used to indicate that someone is marked
+	var/mutable_appearance/moon_insanity_overlay
+	/// icon file for the overlay
+	var/effect_icon = 'icons/effects/eldritch.dmi'
+	/// icon state for the overlay
+	var/effect_icon_state = "moon_insanity_overlay"
+
+/atom/movable/screen/alert/status_effect/moon_converted
+	name = "Moon Converted"
+	desc = "THEY LIE, THEY ALL LIE!!! SLAY THEM!!! BURN THEM!!! MAKE THEM SEE THE TRUTH!!!"
+	icon_state = "lastresort"
+
+/datum/status_effect/moon_converted/on_creation()
+	moon_insanity_overlay = mutable_appearance(effect_icon, effect_icon_state, ABOVE_MOB_LAYER)
+	return ..()
+
+/datum/status_effect/moon_converted/Destroy()
+	QDEL_NULL(moon_insanity_overlay)
+	return ..()
+
+/datum/status_effect/moon_converted/on_apply()
+	RegisterSignal (owner, COMSIG_MOB_APPLY_DAMAGE, PROC_REF(on_damaged))
+	// Healing based on how low their sanity is
+	owner.adjustBruteLoss(-150+owner.mob_mood.sanity)
+	owner.adjustFireLoss(-150+owner.mob_mood.sanity)
+	owner.balloon_alert(owner, "they lie..THEY ALL LIE!!!")
+	owner.AdjustUnconscious(7 SECONDS, ignore_canstun = FALSE)
+	ADD_TRAIT(owner, TRAIT_MUTE, type)
+	RegisterSignal(owner, COMSIG_ATOM_UPDATE_OVERLAYS, PROC_REF(update_owner_overlay))
+	owner.update_appearance(UPDATE_OVERLAYS)
+	owner.cause_hallucination( \
+		/datum/hallucination/delusion/preset/moon, \
+		"[id] status effect", \
+		duration = duration, \
+		affects_us = FALSE, \
+		affects_others = TRUE, \
+	)
+	return TRUE
+
+/datum/status_effect/moon_converted/proc/on_damaged(datum/source, damage, damagetype)
+	SIGNAL_HANDLER
+
+	// Stamina damage is funky so we will ignore it
+	if(damagetype == STAMINA)
+		return
+
+	// Adds damage to the damage sustained
+	damage_sustained += damage
+
+	// If the damage_sustained is less than 75 don't remove the status effect
+	if (damage_sustained<75)
+		return
+	// Remove the status effect
+	owner.remove_status_effect(src)
+
+/datum/status_effect/moon_converted/proc/update_owner_overlay(atom/source, list/overlays)
+	SIGNAL_HANDLER
+
+	overlays += moon_insanity_overlay
+
+/datum/status_effect/moon_converted/on_remove()
+	to_chat(owner, span_notice("<span class='warning'>Your mind is cleared from the effect of the manus, your alligiences are as they were before</span>"))
+	owner.balloon_alert(owner, "your mind clears and you return to normal!")
```

Comment:
Either have a to chat or a balloon alert, not both

Reply:
Picked one